presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -489,6 +490,18 @@ public boolean isLegacyMapSubscript()
         return legacyMapSubscript;
     }
 
+    public boolean isLegacyRowToJsonCast()
+    {
+        return legacyRowToJsonCast;
+    }
+
+    @Config("deprecated.legacy-row-to-json-cast")
+    public FeaturesConfig setLegacyRowToJsonCast(boolean legacyRowToJsonCast)
```

Comment:
This is the part which changes the default to be the new json converted which makes it a breaking change. We can change the default to use legacy cast, if needed